vsintegration/src/FSharp.Editor/Commands/XmlDocCommandService.fs

```diff
@@ -24,12 +24,12 @@ type internal XmlDocCommandFilter
         workspace: VisualStudioWorkspace
      ) =
 
-    let document =
-        // There may be multiple documents with the same file path.
-        // However, for the purpose of generating XmlDoc comments, it is ok to keep only the first document.
-        lazy(match workspace.CurrentSolution.GetDocumentIdsWithFilePath(filePath) |> Seq.toList with
-             | [] -> None
-             | documentId :: _ -> Some (workspace.CurrentSolution.GetDocument documentId))
+    let getLastDocument () =
+        // There may be multiple documents with the same file path, due to versioning .
+        // However, for the purpose of generating XmlDoc comments, it is ok to keep only the last document, since they're versioned.
+        match workspace.CurrentSolution.GetDocumentIdsWithFilePath(filePath) |> Seq.toList |> List.rev with
```

Comment:
Why instead not something like Seq.tryLast or Seq.tryHead?

Reply:
Dunno, didn't want to change much.